.github/steps/-step.txt

```diff
@@ -1 +1 @@
-2
```

Comment:
Introduction to GitHub - reviewing the Pull.

Reply:
A bit lost after the Commit changes but luckily copied the instructions. Should be okay to move forward.